index.html

```diff
@@ -685,6 +714,22 @@ <h3>Converting between <code>tiltX</code> / <code>tiltY</code> and <code>altitud
 </pre>
             </section>
         </section>
+        <section>
+            <h2><code>DeviceProperties</code> interface</h2>
+            <pre class="idl">
+                [Exposed=Window]
+                interface DeviceProperties {
+                    constructor(long uniqueId);
+                    readonly attribute long uniqueId;
+                };
+            </pre>
+            <dl data-dfn-for="DeviceProperties" data-link-for="DeviceProperties">
+                <dt><dfn>uniqueId</dfn></dt>
+                    <dd>
+                        <p>A unique identifier for the pointing device. Pointer events generated from the same physical device will have the same <code>uniqueId</code> for the browsing session. A generic <code>uniqueId</code> value of <code>1</code> MUST be reserved for events generated by the primary mouse device. The <code>uniqueId</code> value of <code>-1</code> MUST be reserved and used to indicate events whose generating device could not be identified. Like <a href="#dom-pointerevent-pointerid">pointerId</a>, to minimize the chance of fingerprinting and tracking across different pages or domains, the <code>uniqueId</code> MUST only be associated explicitly with that particular pointing device for the lifetime of the page / session, and a new randomized <code>uniqueId</code> MUST be chosen the next time that particular pointing device is used again in a new session.</p>
```

Comment:
I think this needs to explain that same physical device in fact means that if the hardware supports multiple pointers they must only get a uniqueId if those pointers are uniquely identifiable over the session.

Reply:
I've tried to address this feedback with updated verbiage. PTAL again!